src/tools/dbimport/dbimport.conf.dist

```diff
@@ -159,6 +159,17 @@ CharacterDatabase.SynchThreads = 1
 
 Updates.EnableDatabases = 7
 
+#
+#   Updates.AllowedModules
+#       Description: A list of modules that are allowed to be updated automatically by the DBImport tool.
+#                    If the list is empty, no modules are allowed to automatically update. (current behavior)
+#       Default:     "" - (No modules are allowed)
+#
+#       Example:     "mod_name,mod_name2,mod_name3" (selected modules)
+#                     "all" - (All modules are allowed)
+#
+Updates.AllowedModules = ""
```

Comment:
I would put `"all"` as default

Reply:
That was my initial thought as well, in testing though I did encounter modules that break the build if they have already been applied.  I was trying to keep it as backwards compatible as possible. Maybe there is a way to figure out a 1 time initial fix for people, document it, then convert it over to all in the future?

Reply:
I would put "all" and then fix the related modules that break the build 
I can fix them if the build does not pass

Reply:
Ok I will do that.